src/retrace/retrace.py

```diff
@@ -1760,7 +1760,14 @@ def prepare_debuginfo(self, vmcore, chroot=None, kernelver=None, crash_cmd=["cra
                               stdin=PIPE, stdout=PIPE, stderr=null)
         else:
             child = Popen(crash_cmd + ["-s", vmcore, vmlinux], stdin=PIPE, stdout=PIPE, stderr=STDOUT)
-        stdout = child.communicate("mod\nquit")[0]
+        try:
+            t = 3600
+            if CONFIG["ProcessCommunicateTimeout"]:
+                t = CONFIG["ProcessCommunicateTimeout"]
+            stdout = child.communicate("mod\nquit", timeout=t)[0]
+        except:
+            child.kill()
+            raise Exception, "WARNING: Possible vmcore memory corruption or damaged file - crash 'mod' command exceeded " + str(t) + " second timeout"
```

Comment:
In Python3 the syntax is:
```python3
raise Exception(...)
```

Can you also make the line fit ~120 characters max?